compatibility-verifier/sample-test-suite/config/feature-test-2-realtime.json

```diff
@@ -41,7 +41,7 @@
     "sortedColumn": [],
     "streamConfigs": {
       "streamType": "kafka",
-      "stream.kafka.topic.name": "PinotRealtimeFeatureTest1Event",
+      "stream.kafka.topic.name": "PinotRealtimeFeatureTest2Event",
```

Comment:
Please add the segment completion criteria here.
`
"realtime.segment.flush.threshold.size": "63"
"realtime.segment.flush.threshold.time": "1h"
`
Since we are publishing 66 rows, it is best to complete a segment a bit below 66, so that we always complete at least one segment each generation, and also have some left over rows that we need to re-consume in the next generation.

Reply:
updated!